Tests/TuistIntegrationTests/Generator/MultipleConfigurationsIntegrationTests.swift

```diff
@@ -284,12 +284,13 @@ final class MultipleConfigurationsIntegrationTests: XCTestCase {
         return Workspace(name: "Workspace", projects: projects.map { pathTo($0) })
     }
 
-    private func createProject(path: AbsolutePath, settings: Settings, targets: [Target]) -> Project {
+    private func createProject(path: AbsolutePath, settings: Settings, targets: [Target], schemes: [Scheme]) -> Project {
```

Comment:
If scheme is not shared, it will be generated at path : `username.xcuserdatad/schemes`. Do you have a any ideas how we can get Mac OS username?
> **shared** flag within Scheme doesn't seem to toggle the setting in the generated scheme

Updated test coverage setting for TestAction
> **coverage** flag within TestAction doesn't seem to enable test coverage in the generated project